allure-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/ntc/webapi/BasePageTest.java

```diff
@@ -0,0 +1,34 @@
+package com.capgemini.ntc.webapi;
+
+import org.junit.Test;
+
+import com.capgemini.ntc.test.core.BaseTest;
+import com.capgemini.ntc.webapi.core.BaseWebApi;
+
+public class BasePageTest extends BaseTest {
```

Comment:
Just another suggestion. I wouldn't call this class BasePageTest, rather something like: BaseWebApiTest.